### ISSUE TITLE
send email reports

### ISSUE THREAD

**Issue by joshua-8** [brown-SSW/brown-solar-charger#25]
we can discuss changes to this, but I'm thinking something like monthly usage summaries with a dump of that month's detailed logs along with immediate error reports if the station goes offline.

**Comment by joshua-8** [comment]
maybe this can be done by a script connected to a spreadsheet?

**Comment by vskbellala** [comment]
I like the idea! Are you planning to use a mailing list type system like [mailchimp](https://mailchimp.com/) so multiple people can subscribe to it?

**Comment by joshua-8** [comment]
I hadn't thought of that, good idea!

**Comment by joshua-8** [comment]
I have made a couple google groups and set up the sfsw.brown.solar.charger gmail (ask me and I'll send you the password) so that if the station emails itself, the message can be forwarded to people who are subscribed to the group. I made one group for regular (maybe monthly) reports on the station's performance, and another group for people who want to receive alerts if the station detects a technical issue that needs to be fixed.
Both groups can be found by going to google groups logging in with the solar charger email, but since I haven't even started the process of testing the system I recommend that no one subscribe yet unless they're the one working on sending emails, since there will probably be a lot of annoying tests. Much later when everything is running smoothly we can tell everyone that they can subscribe to these groups if they want (and we should add the s4sw email).
